adsmp/tasks.py

```diff
@@ -163,15 +164,15 @@ def task_index_records(bibcodes, force=False, update_solr=True, update_metrics=T
             if update_solr:
                 d = solr_updater.transform_json_record(r)
                 logger.debug('Built SOLR: %s', d)
-                if r.get('solr_checksum', None) != app.checksum(d):
+                if r.get('solr_checksum', None) != app.checksum(d) or ignore_checksums:
```

Comment:
looks great!

small detail, please use `if  ignore_checksums or r.get('solr_checksum', None) != app.checksum(d):` instead

we'll save bit of cpu and it's free